docs/user_manual/components.md

```diff
@@ -181,6 +193,38 @@ It can happen that `tap_min > tap_max`. In this case the winding voltage is decr
 increased.
 ```
 
+#### Electric Model
+`transformer` is described by π model, where z_series and y_shunt can be computed by following equations:
+
+$$
+
+$$
+
+z_base = `sn`/(`u2`^2)
+
+|z_series| = |`uk`|/z_base
+
+Re(z_series) = (`pk`/`sn`)/z_base
+
+Im(z_series) = sign(`uk`) * sqrt(|z_series|^2-Re(z_series)^2)
```

Comment:
From a user perspective, uk is always positive as the transformer must have a positive impedance. (Check the restriction at the components table)

`uk` can be negative theoretically at times for eg, the three winding transformer. The way we model a three winding transformer is by star connected 3 two winding transformers. Then `uk` isconverted from delta to star which might lead to negative value.